### PR TITLE
chore: upgrade react-virtualized

### PR DESCRIPTION
**Pull request by jsomsanith**
**What is the problem this PR is trying to solve?**
Split react 16 branch changes into smaller PRs, to ease the changes and the integration.

**What is the chosen solution to this problem?**
Upgrade react-virtualized
No regression on the components
See react-virtualized changelog, only new features, fixes and a lot of chore (conf, prettier, etc). So no breaking changes.

Need to test is on TFD for log viewer but it should be ok.

**Please check if the PR fulfills these requirements**
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/containers/package.json [hunk 1]
```diff
@@ -89,7 +89,7 @@
     "react-storybook-cmf": "^0.2.0",
     "react-stub-context": "^0.7.0",
     "react-test-renderer": "^15.6.2",
-    "react-virtualized": "9.10.1",
+    "react-virtualized": "9.15.0",
```
Comment (jare-talend) [inline packages/containers/package.json]: Should not you update to the latest version: 9.16.0 released ago 7 days :)

Comment (jsomsanith) [reply packages/containers/package.json]: No breaking change in changelog, let's do it

Comment (jsomsanith) [reply packages/containers/package.json]: In fact it's already on 9.16.1 in version.js ... I just forgot to run it :/

Comment (jsomsanith) [reply packages/containers/package.json]: Done

**Comment by nicomaligne** [comment]
Already in [9.17.1]( https://www.npmjs.com/package/react-virtualized ) didn't check the br.

**Comment by jsomsanith** [comment]
Thanks @nicomaligne, I upgraded
Changelog indicates fixes/feats/optimisations only. I tested it on our component, seams ok

**Pull request merged**